doc/service-tests.md

```diff
@@ -97,7 +97,7 @@ boilerplate:
 'use strict';
```

Comment:
A few lines up from here (line 92) there is a path `service-tests/travis.js` which needs updating, but annoyingly I can't leave a diff comment on that exact line

Reply:
Fixed!